apps/studio/components/interfaces/SQLEditor/SQLEditor.tsx

```diff
@@ -311,13 +311,30 @@ const SQLEditor = () => {
           return toast.error('Unable to run query: Connection string is missing')
         }
 
+        // Remove lines and whitespaces
+        const cleanedSql = sql.trim().replaceAll('\n', ' ').replaceAll(/\s+/g, ' ')
+        // Check if need to auto limit rows
+        const appendAutoLimit =
+          snap.limit > 0 &&
+          cleanedSql.toLowerCase().startsWith('select') &&
+          !cleanedSql.endsWith('limit') &&
+          !cleanedSql.endsWith('limit;') &&
+          !!!cleanedSql.match('limit [0-9]*[;]?$')
```

Comment:
triple bang?!?!

Reply:
hahaha im wondering what would be a cleaner way to do this 😅 
wanted to check for falsey value of that 

maybe `(!!cleanedSql.match(...) === false)` might be better you reckon? for readability

Reply:
i'll change it to just 1 hahaha